spring-integration-http/src/main/java/org/springframework/integration/http/inbound/HttpRequestHandlingEndpointSupport.java

```diff
@@ -148,8 +148,6 @@
 
 	private volatile Map<String, Expression> headerExpressions;
 
-	private volatile boolean shuttingDown;
```

Comment:
OK. Thank you for pointing this out.
I really don't understand this stuff yet...
Will revise what's going on there around.